samples/core-projects/libraries/new-library/test/LibraryTests/project.json

```diff
@@ -1,14 +1,22 @@
 {
-	"commands": {
-		"test":"xunit.runner.dnx"
+	"version": "1.0.0",
+	"buildOptions": {
+		"debugType": "portable"
 	},
+	"testRunner": "xunit",
 	"frameworks": {
-		"dnxcore50":{
+		"netcoreapp1.0":{
 			"dependencies": {
-				"Library":"",
-				"System.Runtime":"4.0.0-rc1-*",
-				"xunit":"2.1.0",
-				"xunit.runner.dnx": "2.1.0-rc1-*"
+				"Library":{
+					"version": "1.0.0",
```

Comment:
I thought we recommended omitting the version element here for project reference targets. Did you mean a specific version?